Course/Course/Data/CourseStorage.swift

```diff
@@ -11,6 +11,7 @@ import Core
 public protocol CourseStorage {
     var allowedDownloadLargeFile: Bool? { get set }
     var userSettings: UserSettings? { get set }
+    var videoPlaybackSpeed: Float? {get set}
```

Comment:
Maybe better to move it to `UserSettings` where we have `streamingQuality` option?

Reply:
We don't have a separate setting for this at the profile/settings level. It can only be changed in the video player. I don't think we need to mix this with the settings object. Thank you

Reply:
It would be nice if we could group all the user settings regardless of their place of usage.

Reply:
done. please re-check